### DISCUSSION TITLE
多个dumi工程如何共享node_modules？

### DISCUSSION THREAD

**Discussion by fanhualei** [umijs/dumi#936] [Q&A]
多个dumi工程如何共享node_modules？

我把其中一个的node_modules，放到这些工程的父目录中。 yarn start 出现错误

```sh
yarn start
yarn run v1.22.17
$ dumi dev
/bin/sh: 1: dumi: not found
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

如果我在子目录中执行`yarn install`，就会在子目录生成`node_modules`

**Comment by leohxj** [comment]
可以看下 monorepo 的工程化方案, 比如使用 pnpm workspace, 这里有个示例: https://github.com/MIchaelJier/dumi-pnpm-template
